q2_phylogeny/plugin_setup.py

```diff
@@ -193,7 +193,7 @@
     inputs={'alignment': FeatureData[AlignedSequence]},
     parameters={
             'seed': Int,
-            'n_threads': Int % Range(1, None),
+            'n_threads': Int,
```

Comment:
Ah, missed this early, the iqtree docs indicate that this is multicore processing, not multi-threaded. We should probably call this parameter `n_cores`. The support text could say:

> The number of CPU cores to use for processing. Use '0' to let IQ-TREE automatically determine the optimal number of cores to use.

Reply:
Right! I fixed this throughout the code / test code.